internal/cli/users.go

```diff
@@ -205,6 +206,7 @@ func searchUsersCmd(cli *cli) *cobra.Command {
 	}
 
 	cmd.Flags().BoolVar(&cli.json, "json", false, "Output in json format.")
+	cmd.Flags().BoolVar(&cli.csv, "csv", false, "Output in csv format.")
```

Comment:
**Question:** Wouldn't this need to be added to the rest of the commands to be applied cli-wide?

Reply:
Yes, I can do that